lib/blackjack_score.rb

```diff
@@ -1,24 +1,51 @@
 class BlackjackScore
 
   def get_hand_value(hand)
-    get_card_value(hand[0])
-  end
+    card_values = []
+    hand.each do |card|
+      value = get_card_value(card)
+      card_values.push(value)
+    end
 
+    sum = card_values.reduce(:+)
+    if sum == -1
+      sum
+    elsif sum > 21 && card_values.include?(11)
+    value_occurences = get_occurence_number(card_values)
```

Comment:
This block is a little confusing to me. Can you extract it to a method and give it an appropriate name?